packages/core/contracts/oracle/implementation/SlashingLibrary.sol

```diff
@@ -15,7 +15,9 @@ contract SlashingLibrary {
         uint256 totalCorrectVotes
     ) public pure returns (uint256) {
         // This number is equal to the slash amount needed to cancel an APY of 20%
-        // if 10 votes are cast each month for a year. 0.2/(10*12)= ~0.0016
+        // if 10 votes are cast each month for a year.  1 - (1 / 1.2)**(1/120) = ~0.0016
```

Comment:
The result of this function is closer to 0.0015 than 0.0016, should we change the constant or something?

Reply:
We have 13 tests that rely on this `0.0016` to be correct: the values are hardcoded to showcase the maths; we can change it, but I'm not sure what the added value is if we plan to replace it with a simpler value later on, such as `0.001` 

I am happy to change them if you think we should change to 0.0015. If so, maybe I should remove the hardcoded values from the tests? cc @chrismaree

Reply:
we should not change this. leave it at 0.0016, the time taken to update all the tests and the comments is not worth it. this magic number was chosen as something approximately useful or close to what we'd like. it was not meant to be accurate or precise; we'll likely start at something like `0.001` when we deploy the contracts so it's really easy to reason about

(note when I set up the tests I intentionally did not choose `0.001` as using perfect round numbers can obfuscate complexity in computations as you end up scaling things up or down by a factor which can cause issues when reasoning about things).